src/main/java/io/ddavison/conductor/util/ScreenShotUtil.java

```diff
@@ -26,10 +26,12 @@ public static void take(Locomotive locomotive, String path, String testName) {
     }
 
     private static void writeFile(TakesScreenshot takesScreenshot, String filePathAndName) {
-        try {
-            FileUtils.copyFile(takesScreenshot.getScreenshotAs(OutputType.FILE), new File(filePathAndName));
-        } catch (IOException e) {
-            e.printStackTrace();
+        if (takesScreenshot != null && filePathAndName != null) {
+            try {
+                FileUtils.copyFile(takesScreenshot.getScreenshotAs(OutputType.FILE), new File(filePathAndName));
+            } catch (IOException e) {
+                e.printStackTrace();
```

Comment:
can we do this through a logging library, like `Logger.error(e)`?

Reply:
Done.